pkg/dynatrace/dynatrace.go

```diff
@@ -151,6 +151,31 @@ func (dynatraceApiClient *DynatraceApiClient) MakeRequest(method, path string, r
 	return dynatraceApiClient.httpClient.Do(req)
 }
 
+func (dynatraceApiClient *DynatraceApiClient) GetListDynatraceHttpMonitors(clusterId string) (ExistsHttpMonitorInDynatraceResponse, error) {
```

Comment:
You could also have this return a pointer `*ExistsHttpMonitorInDynatraceResponse` then in your error handling, return `nil` when returning an error. It's semantic and you can leave it as is, but that's another common pattern.

Reply:
@joshbranham  do you mind checking that I updated this correctly as you suggested?